fedimint-client/src/client/builder.rs

```diff
@@ -1063,30 +1038,43 @@ impl ClientPreview {
     ) -> anyhow::Result<ClientHandle> {
         let pre_root_secret = pre_root_secret.to_inner(self.config.calculate_federation_id());
 
-        let backup = if let Some(backup) = custom_backup {
-            Some(backup)
-        } else {
-            self.inner
-                .download_backup_from_federation(
-                    pre_root_secret.clone(),
-                    &self.config,
-                    self.api_secret.clone(),
-                )
-                .await?
-        };
-
         let client = self
             .inner
             .init(
                 pre_root_secret,
                 self.config,
                 self.api_secret,
                 InitMode::Recover {
-                    snapshot: backup.clone(),
+                    snapshot: custom_backup.clone(),
```

Comment:
nit: rename to `backup` since it's always required now if recovery is to use a backup